functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -1,24 +1,67 @@
 import logging
-
+import pyodbc
+import os
 import azure.functions as func
+import datetime
+import json
+import time
+
 
 
+# to handle datetime with JSON
+def default(o):

```

Comment:
As a general note, you do have some commenting, but I think it'd be really great to add more, especially around some areas I'll note below. For this particular section, I think that "o" is probably not the best parameter name possible. Maybe something more specific would be nice?